docs/user-guide.md

```diff
@@ -37,13 +37,18 @@ This guide provides detailed instructions on how to import and configure your Li
 
     - Liberty Maven Plugin -> 3.7.1
 
-    - Liberty Gradle Plugin -> 3.5.1
+    - Liberty Gradle Plugin -> 3.6.1
 
 ### Settings
 
-Liberty Tools for IntelliJ IDEA honours  the following external settings:
-- Maven home path set in the **Preferences > Build, Execution, Deployment > Build Tools > Maven** window when running Liberty dev mode on Maven projects.
-- Gradle JVM set in the **Preferences > Build, Execution, Deployment > Build Tools > Gradle** window when running Liberty dev mode on Gradle projects.
+Liberty Tools for IntelliJ IDEA honors the following settings:
+
+#### IntelliJ settings:
+- The Maven home path that is set in the **Settings > Build, Execution, Deployment > Build Tools > Maven** window when you run Liberty dev mode on Maven projects.
+- The Gradle Distribution that is set in the **Settings > Build, Execution, Deployment > Build Tools > Gradle** window when you run Liberty dev mode on Gradle projects.
```

Comment:
I noticed that depending on the IntelliJ UI (Classic vs New), it's called either Preferences or Settings. Should we make a note of that?
Not sure if just calling it **Settings/Preferences** is ok, or maybe stick with **Preferences** and add a note saying it may be called **Settings** if using the new UI.

Reply:
@evie-lau what are the two views called w/in Intellij- Classic and New? Is that version-dependent or an option where the user chooses which UI to use, regardless of their version?